### DISCUSSION TITLE
Cannot resolve permissions issue

### DISCUSSION THREAD

**Discussion by tome88** [jellyfin/jellyfin#9493] [Q&A] [answered]
Hello, getting the following error message when downloading series:

![Untitled1](https://user-images.githubusercontent.com/122657616/225208666-996d1823-811b-4d90-bc86-fe479f23f281.jpg)

however it seems all my permissions are good?

<img width="702" alt="untitled3" src="https://user-images.githubusercontent.com/122657616/225208819-6ffe86e1-abfb-414c-b2a3-7d4c8c4b1e28.png">

would anyone know what I've got wrong?

**Comment by nielsvanvelzen** [comment, marked as answer]
This issue doesn't seem to be caused by Jellyfin, the exception is from another tool (NzbDrone).

**Reply by tome88** [reply]
thanks for the prompt reply @nielsvanvelzen
